### PR TITLE
skeleton of attack function

### PR DESCRIPTION
**Pull request by morecallan**

### CODE REVIEW

**Comment by devoisaurus** [comment]
👍

**Comment by csbarrett1** [comment]
👍

File: javascripts/attack.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  return Math.floor(Math.random() * (max - min)) + min;
+}
+
+function attack(attacker, opponent) {
+    //Give attacker "readyToAttack" class and remove "standby" class
+    
+    //Combatant's attack score is caluclated
+    var damageToOpponentHealth = calculateAttackDamage(attacker);
+    //Opponent's health is reduced by attack score
+    opponent.health = opponent - damageToOpponentHealth;
```
Comment (csbarrett1) [inline javascripts/attack.js]: opponent.health

File: javascripts/attack.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    
+    //Remove attacker "readyToAttack" class and add "standby" class
+    
+    break;
+}
+
+//This is what happens if attack button is pressed
+function attackSequence(human, monster) {
+    //Hide attack button
+    attack(human, monster);
```
Comment (csbarrett1) [inline javascripts/attack.js:47]: delay between attacks

**Pull request merged**